### PR TITLE
feat: Suggest adding collection contract address as part of the TP urn

### LINKED ISSUE #1838: We should support at least an address for the collection name for thirdparties an encourage to use it
@nachomazzara commented on [Fri Mar 11 2022](https://github.com/decentraland/account/issues/196)

Can we add a message saying we encourage the creator to use the collection address as the id?

### PR DESCRIPTION
**Pull request by juanmahidalgo**
## Modal before this changes: 

image

## After these changes:

image

Closes #1838

### CODE REVIEW

File: src/modules/collection/types.ts [hunk 1]
```diff
@@ -41,3 +41,4 @@ export type Access = {
 }
 
 export const COLLECTION_NAME_MAX_LENGTH = 32
+export const TP_COLLECTION_NAME_MAX_LENGTH = 64
```
Comment (nicosantangelo) [inline src/modules/collection/types.ts:44]: Should we be using this length on the EditCollectionNameModal? it was generic up until this point but it uses `COLLECTION_NAME_MAX_LENGTH`

Comment (nicosantangelo) [reply src/modules/collection/types.ts:44]: The UI scares me a bit, things are a bit cramped up:

Current:
Screen Shot 2022-03-14 at 12 24 53


Longer:

image

Comment (juanmahidalgo) [reply src/modules/collection/types.ts:44]: Good call! I updated the modal name to use `TP_COLLECTION_NAME_MAX_LENGTH` as the max length for TPs. 
Also, I updated the styles so the name container doesn't overflow when it's too long:

image

Comment (nicosantangelo) [reply src/modules/collection/types.ts:44]: telegram-cloud-photo-size-1-5010286617772009953-y

**Review by nicosantangelo — APPROVED** [review]

**Pull request merged**